packages/localstorage-adapter/modules/adapter/adapter.ts

```diff
@@ -114,22 +115,25 @@ export const updateFlags = (flags: TFlags) => {
   adapterState.emitter.emit('flagsStateChange', nextFlags);
 };
 
+const didFlagsChange = (nextFlags: TFlags) => {
+  const previousFlags = adapterState.flags;
```

Comment:
`adapterState.flags` is never set, so the object is always empty.

It looks like it should be set in `updateFlags`, `subscribeToFlagsChanges`, and `configure` right before each instance of `adapterState.emitter.emit('flagsStateChange', ... )`.